src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -4620,9 +4631,8 @@ bool StorageReplicatedMergeTree::optimize(
 
             if (create_result == CreateMergeEntryResult::MissingPart)
             {
-                String message = "Can't create merge queue node in ZooKeeper, because some parts are missing";
-                LOG_TRACE(log, fmt::runtime(message));
-                return handle_noop(message);
+                constexpr const char * message_fmt = "Can't create merge queue node in ZooKeeper, because some parts are missing";
```

Comment:
Noncritical but for constants always prefer `static constexpr` over `constexpr`.